underlay/src/main/proto/criteriaselector/dataschema/output_unfiltered.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package tanagra.dataschema;
+
+option go_package = "github.com/DataBiosphere/tanagra/criteriaselector/dataschemapb";
+option java_package = "bio.terra.tanagra.proto.criteriaselector.dataschema";
+option java_outer_classname = "DTOutputUnfiltered";
+
+message OutputUnfiltered {
+  repeated string entities = 1;
```

Comment:
This is a subset of the entities list in the config, is that right? So the filter builder class should only look at the data and ignore the config.

Reply:
No, they're for the same purpose. I had just already implemented it as config before we changed it to data so I just left it in. I implemented it to use the data copy if set and otherwise the config. The config version is currently never used anywhere and we can change it as we flesh out the path forward for these prepackaged feature sets.

Reply:
OK, sounds good. Doesn't have to be here, but this would be very helpful to have before hooking up the backend config.